components/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/dao/AppMDAO.java

```diff
@@ -439,14 +465,96 @@ public void saveBusinessOwner(BusinessOwner businessOwner){
                 try {
                     connection.rollback();
                 } catch (SQLException e1) {
-                    log.error("Failed to rollback the add entitlement policy partial with name : ", e1);
+                    handleException("Failed the rollback of save business owner with name : " + businessOwnerName, e1);
                 }
             }
         } finally {
             APIMgtDBUtil.closeAllConnections(statementToInserBusinessOwner, connection, null);
         }
+        return businessOwnerId;
     }
 
+    /**
+     * Search business owners.
```

Comment:
Java doc comments issues and formatting issues are fixed in https://github.com/wso2/carbon-appmgt/pull/539/files